README.md

```diff
@@ -1,75 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'), kindly keep this in your mind for future projects